app/serializers/submission_serializer/sections/provider_details.rb

```diff
@@ -0,0 +1,23 @@
+module SubmissionSerializer
+  module Sections
+    class ProviderDetails < Sections::BaseSection
```

Comment:
We can clean up the example code here, and `min_version` because `provider_details` is now a required object in the schema. But for now it can be an empty object (`{}`)